src/pip/_internal/network/xmlrpc.py

```diff
@@ -21,12 +24,14 @@ class PipXmlrpcTransport(xmlrpc_client.Transport):
     """
 
     def __init__(self, index_url, session, use_datetime=False):
+        # type: (str, PipSession, Optional[bool]) -> None
```

Comment:
```suggestion
        # type: (str, PipSession, bool) -> None
```

Reply:
Aah, good catch. Not sure why added `Optional` to a default parametrized bool. Fixed in https://github.com/pypa/pip/pull/8271/commits/65a6152f024de93c91a152847faee925f73c050a